tutorials/plot_artifacts_correction_ica.py

```diff
@@ -236,20 +236,30 @@
 # Indeed it looks like an EOG, also in the average time course.
 #
 # We construct a list where our reference run is the first element. Then we
-# can detect similar components from the other runs using
-# :func:`mne.preprocessing.corrmap`. So our template must be a tuple like
+# can detect similar components from the other runs (the other ICA objects)
+# using :func:`mne.preprocessing.corrmap`. So our template must be a tuple like
 # (reference_run_index, component_index):
 icas = [reference_ica] + icas_from_other_data
 template = (0, eog_inds[0])
 
 ###############################################################################
-# Now we can do the corrmap.
+# Now we can run the CORRMAP algorithm.
 fig_template, fig_detected = corrmap(icas, template=template, label="blinks",
                                      show=True, threshold=.8, ch_type='mag')
 
 ###############################################################################
 # Nice, we have found similar ICs from the other (simulated) runs!
-# This is even nicer if we have 20 or 100 ICA solutions in a list.
+# In this way, you can detect a type of artifact semi-automatically for example
+# for all subjects in a study.
+# The detected template can also be retrieved as an array and stored; this
+# array can be used as an alternative template to
+# :func:`mne.preprocessing.corrmap`.
+eog_component = reference_ica.get_components()[:, eog_inds[0]]
+
+# If you calculate a new ICA solution, you can provide this array instead of
```

Comment:
Ok, I get it now. What about change it a little then: "Later, when you calculate another ICA solution for similar data ..."?